### PR TITLE
10.18.0.203

### PR DESCRIPTION
**Pull request by BeateRixen**

### CODE REVIEW

**Comment by BeateRixen** [comment]
This must be updated when the actual maintenance release version is provided

File: content/release-10-18-0/applications-10-18-0-bundle/applications-10-18-0-229.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+<td>Cockpit</td>
+<td>The <b>Copy dashboard</b> button is now disabled when the user does not have the respective permissions.</td>
+<td>MTM-53334</td>
+<td>10.18.0.140</td>
+<td>UI</td>
+</tr>
+
+<tr>
+<td>Cockpit</td>
+<td>In the "Data points graph" widget, when realtime is enabled and users double-click on a chart, the time range expands now and the data for the current range is shown.</td>
```
Comment (rssag) [inline content/release-10-18-0/applications-10-18-0-bundle/applications-10-18-0-229.md:54]:
```suggestion
<td>In the "Data points graph" widget, when realtime is enabled and users double-click on a chart, the time range now expands and the data for the current range is shown.</td>

```

File: content/release-10-18-0/applications-10-18-0-bundle/applications-10-18-0-229.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+<td>Cockpit</td>
+<td>The <b>Add report</b> button is now disabled when the user does not have sufficient permissions.</td>
+<td>MTM-55195</td>
+<td>10.18.0.130</td>
+<td>UI</td>
+</tr>
+
+<tr>
+<td>Java SDK</td>
+<td>Resolved an issue where multiple realtime API subscriptions made in quick succession could cause an invalid subscription state, leading to subscriptions receiving duplicate notifications.</td>
```
Comment (rssag) [inline content/release-10-18-0/applications-10-18-0-bundle/applications-10-18-0-229.md:70]:
```suggestion
<td>Resolved an issue where multiple realtime API subscriptions in quick succession could cause an invalid subscription state, leading to subscriptions receiving duplicate notifications.</td>

```

File: content/release-10-18-0/devices-10-18-0-bundle/devices-10-18-0-229.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+<td>Device management</td>
+<td>Fixed an issue in the data grid and device grid components where specific string, Boolean, and numeric cell values were wrongly formatted as dates.</td>
+<td>DM-2746</td>
+<td>10.18.0.150</td>
+<td>UI</td>
+</tr>
+
+<tr>
+<td>LWM2M</td>
+<td>Setting up LWM2M agent DTLS connections by DTLS Connection IDs (CID) is now supported. By default this functionality is enabled. If you use LWM2M devices that support connection IDs, contact your platform administrator&nbsp;to apply additional settings or check if this feature is enabled.</td>
```
Comment (rssag) [inline content/release-10-18-0/devices-10-18-0-bundle/devices-10-18-0-229.md:54]:
```suggestion
<td>Setting up LWM2M agent DTLS connections by DTLS Connection IDs (CID) is now supported. By default this functionality is enabled. If you use LWM2M devices that support connection IDs, contact your platform administrator to apply additional settings or check if this feature is enabled.</td>
```

File: content/release-10-18-0/platform-10-18-0-bundle/platform-10-18-0-229.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+<td>Authentication</td>
+<td>When a user logs in using OAI-Secure and a password change is required, a <code>PasswordResetToken</code> is returned in the response header, enabling the password reset.</td>
+<td>MTM-55200</td>
+<td>10.18.0.209</td>
+<td>Core</td>
+</tr>
+
+<tr>
+<td>Messaging Service</td>
+<td>Users of Notifications 2.0 will no longer encounter unequal distribution of notifications from tenant-context subscriptions amongst a set of shared consumers.</td>
```
Comment (rssag) [inline content/release-10-18-0/platform-10-18-0-bundle/platform-10-18-0-229.md:48]:
```suggestion
<td>Users of Notifications 2.0 no longer encounter unequal distribution of notifications from tenant-context subscriptions among a set of shared consumers.</td>

```

File: content/release-10-18-0/platform-10-18-0-bundle/platform-10-18-0-229.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+<td>REST API</td>
+<td>When removing an application that is used in SSO access mappings, the login configuration will be updated accordingly.</td>
+<td>MTM-52943</td>
+<td>10.18.0.200</td>
+<td>Core</td>
+</tr>
+
+<tr>
+<td>REST API</td>
+<td>The Measurement API now accepts leading zeroes provided for measurement values.</td>
```
Comment (rssag) [inline content/release-10-18-0/platform-10-18-0-bundle/platform-10-18-0-229.md:72]:
```suggestion
<td>The Measurement API now accepts leading zeros provided for measurement values.</td>

```

**Review by rssag — CHANGES_REQUESTED** [review]

File: content/release-10-18-0/applications-10-18-0-bundle/applications-10-18-0-229.md (hunk 1 above)
Comment (BeateRixen) [reply content/release-10-18-0/applications-10-18-0-bundle/applications-10-18-0-229.md:54]: Leave as is due to consistency with other releases

**Pull request merged**